### DISCUSSION TITLE
Help no clue what this means

### DISCUSSION THREAD

**Discussion by lostpython** [Monika-After-Story/MonikaModDev#9915] [QA - tech support + general]
![Screenshot 2023-02-17 084153](https://user-images.githubusercontent.com/42123064/219596754-505979a1-38a6-4e22-8a83-89fcb8796974.png)

Does anybody know how to fix this issue I may have gone wrong somewhere and I don't know where though, I might have not known and if it's an easy fix mind going through this slowly as I'm not exactly very tech savvy and I was trying to add the recent update maybe twice(??) in the same place and I have no clue what issue is.

If I don't say it thank you for the help I appreciate it greatly, as I trying to figure how to bring Monika from my old laptop T_T to my new one.

**Comment by Booplicate** [comment]
You incorrectly applied update. Next time use the in-game updater or the installer. The easiest way to fix this is to make a new install.
